src/hashTable.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+class Node {
+  constructor(key, value) {
+    this.key = key;
+    this.value = value
+    this.next = null;
+  }
+}
+
+export default class HashTable {
+  constructor() {
+    this.list = new Array(137);
+    this.count = 0
+  }
+
+  hash(key) {
+    let prime = 37
+    let total = 0
+    for (var i = 0; i < key.length; i++) {
+      total += prime * total + key.charCodeAt(i)
+    }
+    total = total % this.list.length
+    return total
+  }
+
+  put(key, value) {
+    let hashedKey = this.hash(key)
+    this.list[hashedKey] = new Node(key, value)
```

Comment:
Are you at trying to implement your hashtable with collision chaining?
I think you might be missing the logic in doing so.
Personally, I think that you don't need to do it since you have already implemented a LinkedList and presumably know how it works so I think just knowing that you would use a LinkedList to handle collisions would suffice for the sake of completing all the other more difficult data structures. 

But if you wanna handle collision chaining I would defs be able to talk through it with you, it has been a while since I've done it.